maxim/models/prompt.py

```
@@ -5,113 +5,206 @@
 @dataclass
-class FunctionCall():
+class FunctionCall:
+    """
+    This class represents a function call.
+    """
+
     name: str
     arguments: str
 
     @staticmethod
     def from_dict(obj: Dict[str, Any]):
```

Ruff (0.11.9)

Missing return type annotation for staticmethod `from_dict`

(ANN205)